frontend/src/components/WofPopper.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+    <div class="popper">
+        <div class="popper-content">
+            <slot></slot> 
+        </div>
+    </div>
+</template>
+
+<script>
+export default{
+    name: "WofPopper"
```

Comment:
Add isOpen prop with type of Boolean and default false. We want to display it conditionally based on prop value.